src/partials/Banner.svelte

```diff
@@ -134,6 +134,10 @@
         <!-- optional slot for a CTA button -->
         <slot name="button" />
       </div>
+      <div class="bx--col-lg-6">
```

Comment:
Update class to `bx--col-lg-4 bx--offset-lg-2`. This will make the column width smaller and push it to the right. With that change you won't need to hard code width for the `cta-card` slot.